proto/src/determined/container/v1/container.proto

```diff
@@ -38,4 +38,10 @@ message Container {
   State state = 3;
   // A list of devices that is being used by this container.
   repeated determined.device.v1.Device devices = 4;
+  // EE user has no RBAC viewing permissions.
```

Comment:
```suggestion
  // User has insufficient permissions to view this container's details
```
Isn't this abstractly possible for any authz implementation, including but not limited to RBAC?

Reply:
Yes, good point! Changed this comment